File: OsmAnd-java/src/main/java/net/osmand/data/QuadPointDouble.java
package net.osmand.data;

public class QuadPointDouble {
	public double x;
	public double y;

	public QuadPointDouble() {
	}
	
	public QuadPointDouble(double x, double y) {
		this.x = x;
		this.y = y;
	}

	public QuadPointDouble(QuadPointDouble a) {
		this(a.x, a.y);
	}

	public void set(double x, double y) {
		this.x = x;
		this.y = y;
	}

}

File: OsmAnd-shared/src/commonMain/kotlin/net/osmand/shared/data/QuadPointDouble.kt
package net.osmand.shared.data

class QuadPointDouble {
	var x: Double
	var y: Double

	constructor() {
		x = 0.0
		y = 0.0
	}

	constructor(x: Double, y: Double) {
		this.x = x
		this.y = y
	}

	constructor(a: QuadPointDouble) {
		this.x = a.x
		this.y = a.y
	}

	fun set(x: Double, y: Double) {
		this.x = x
		this.y = y
	}
}
